docs/css-prop.mdx

```diff
@@ -14,10 +14,35 @@ There are 2 ways to get started with the `css` prop.
 Both methods result in the same compiled code.
 After adding the preset or setting the pragma as a comment, compiled jsx code will use emotion's `jsx` function instead of `React.createElement`.
 
-|        | Input                      | Output                                              |
-| ------ | -------------------------- | --------------------------------------------------- |
-| Before | `<img src="avatar.png" />` | `React.createElement('img', { src: 'avatar.png' })` |
-| After  | `<img src="avatar.png" />` | `jsx('img', { src: 'avatar.png' })`                 |
+<table>
```

Comment:
q: any particular problems with markdown tables that were hard to fix?

Reply:
Markdown tables are part of GitHub flavored markdown so supporting them requires an extra plugin. I actually had that working, but I ultimately decided it was simpler to just write the 1 table this affected to HTML so we didn't have to use yet another plugin.

Reply:
I don't feel super strongly about this, but I got used to writing tables in markdown this way (TIL that this syntax is part of GFM). I'm a bit worried that I might add a table using this syntax at some point in time and it won't work.

Whatever you decide on this one is good - if adding a plugin for this is straightforward then I would go for it (I would try to stay 100% compliant with GFM, just to be consistent with GitHub as those markdown files are rendered here anyway).